.github/workflows/deployment_cd_test_prod.yml

```diff
@@ -0,0 +1,63 @@
+name: Deployment pipeline for post-award services
+run-name: Deploy branch ${{ github.ref_name }} to test and prod (${{ format('SHA:{0}', github.sha) }})
+on:
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: deploy-test
+  cancel-in-progress: false
+
+jobs:
+  paketo_build:
+    permissions:
+      packages: write
+    concurrency: build-${{ github.sha }}
+    uses: communitiesuk/funding-service-design-workflows/.github/workflows/package.yml@main
+    with:
+      assets_required: true
+      version_to_build: ${{ github.sha }}
+      owner: ${{ github.repository_owner }}
+      application: ${{ github.event.repository.name }}
+
+  copilot_test_deploy:
+    needs: [ paketo_build ]
+    strategy:
+      matrix:
+        include:
+          - deployment: post-award
+            command: svc
+          - deployment: post-award-celery
+            command: svc
+          - deployment: download-report
+            command: job
+    uses: ./.github/workflows/aws_copilot_deployment.yml
+    with:
+      copilot_environment: test
+      github_sha: ${{ github.sha }}
+      deployment: ${{ matrix.deployment }}
+      command: ${{ matrix.command }}
+      IMAGE_LOCATION: ghcr.io/${{ github.repository_owner }}/${{ github.event.repository.name }}:${{ github.sha }}
```

Comment:
If this is the same in both environments, should it not be an input and instead just be built in `aws_copilot_deployment.yml` job?

Reply:
Yes you're right, the called workflow should have access to the github context so this should be doable with the change above! I've pushed a commit that makes these changes.